[2024-08-25 17:42:28,222] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-05T00:00:00+00:00 [queued]>
[2024-08-25 17:42:28,227] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-05T00:00:00+00:00 [queued]>
[2024-08-25 17:42:28,227] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-08-25 17:42:28,227] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-08-25 17:42:28,227] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-08-25 17:42:28,241] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-12-05 00:00:00+00:00
[2024-08-25 17:42:28,244] {standard_task_runner.py:52} INFO - Started process 162112 to run task
[2024-08-25 17:42:28,246] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2022-12-05T00:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpy2cko_cy', '--error-file', '/tmp/tmpos131h8b']
[2024-08-25 17:42:28,246] {standard_task_runner.py:80} INFO - Job 52: Subtask extrai_dados
[2024-08-25 17:42:28,282] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-05T00:00:00+00:00 [running]> on host adrilucy.
[2024-08-25 17:42:28,326] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-05T00:00:00+00:00
[2024-08-25 17:42:29,184] {python.py:173} INFO - Done. Returned value was: None
[2024-08-25 17:42:29,190] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20221205T000000, start_date=20240825T204228, end_date=20240825T204229
[2024-08-25 17:42:29,227] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-08-25 17:42:29,239] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
